tf-build-tools/tf-maven-plugin/src/test/java/com/deliveredtechnologies/maven/terraform/TerraformPackageTest.java

```diff
@@ -93,8 +103,10 @@ public void packageWithFatTarPackagesTfModulesInsideTfRootInTheTargetDir() throw
         tarEntryNames.add(entry.getName());
       }
 
-      Assert.assertEquals(5, count);
+      Assert.assertEquals(10, count);
       Assert.assertTrue(tarEntryNames.contains("main.tf"));
+      Assert.assertTrue(tarEntryNames.contains("dev1.auto.tfvars"));
```

Comment:
I think this format of the assertion produces better output when it fails
Assert.assertThat(tfCommand, CoreMatchers.containsString("-json "));

Reply:
Thanks Rob. I think TarEntryNames type may not be supported in CoreMatches.